Use focus manager for navigation context menus

mtsgrd opened the pull request:
Removes custom keyboard navigation for context menus, and replaces it 
with the new focus manager functionality. Note that ContextMenuItem 
contains two focusables, one for navigation and one to support f-keys.

mtsgrd commented:
https://github.com/user-attachments/assets/d0eba3e0-186e-498a-a332-c76912813641

@PavelLaptev could you have a look at this and let me know what you think?

PavelLaptev commented:
> To me it looks like I included a change with `outline: none` to exactly prevent the double border. Could you be needing to recompile?

@mtsgrd sorry, I was looking at the video you attached.

mtsgrd commented:
I'll investigate!

mtsgrd committed:
Use focus manager for navigation context menus

Removes custom keyboard navigation for context menus, and replaces it 
with the new focus manager functionality. Note that ContextMenuItem 
contains two focusables, one for navigation and one to support f-keys.

mtsgrd commented:
@PavelLaptev just fyi, I still intend to land something like this but I'm currently trying to work through an inconsistency between Chrome and Safari wrt to `:focus-visible`. It's proving to be a bit tricky.
